policy/custom/assignments/AKS.bicep

```diff
@@ -76,6 +76,22 @@ resource podSecurityBaselineStandardsPolicySetAssignment 'Microsoft.Authorizatio
   location: deployment().location
 }
 
+resource ClustersShouldUseInternalLoadBalancers 'Microsoft.Authorization/policyAssignments@2020-03-01' = {
```

Comment:
We are using Policy Sets/Initiative to reduce the number of individual assignments.  Can you move this policy assignment to the AKS policy set definition instead?  This is located in `policy/custom/definitions/policyset/AKS.bicep`

Reply:
@SenthuranSivananthan, I moved the policy assignment accordingly! Thank you for your insights!!